cmd/cmd/userDelete.go

```diff
@@ -0,0 +1,91 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+    "fmt"
+    "github.com/spf13/cobra"
+    "github.com/wso2/product-mi-tooling/cmd/utils"
+    "strings"
+)
+
+// Delete user command related usage info
+const deleteUserCmdLiteral = "delete"
```

Comment:
https://github.com/wso2/product-mi-tooling/blob/master/cmd/cmd/remoteRemove.go#L28
We have defined 'remove' as the command for removing a remote a server. IMO, we should maintain one keyword for consistency. 'delete' is better than 'remove'. Either way it should be consistent. WDYT?

Reply:
+1 fixed